sql/handler.h

```diff
@@ -1836,6 +1836,8 @@ typedef bool (*rotate_encryption_master_key_t)(void);
 */
 using fix_tablespaces_empty_uuid_t = bool (*)(void);
 
+using fix_default_table_encryption_t = void (*)(ulong);
```

Comment:
Is the new handlerton added to handle encryption attribute during upgrade?

Reply:
no, it is for handling changing default_table_encryption variable. The encryption threads in the background check this variable repeatedly and I need to set fil_crypt_threads_event when it is set. Thus I need this handler anyways - and I am also reusing it for changing local copy of this variable inside innodb - so to not have to reach out to the server all the time.

Reply:
can you please document in the code it is used for encryption threads only?